hack/make/.integration-daemon-start

```diff
@@ -96,7 +96,10 @@ if [ -z "$DOCKER_TEST_HOST" ]; then
 	fi
 
 	# "pwd" tricks to make sure $DEST is an absolute path, not a relative one
-	export DOCKER_HOST="unix://$(cd "$DEST" && pwd)/docker.sock"
+	sock_dir="/run/docker/test/$(basename \"${DEST}\")"
```

Comment:
Hmm, seeing a failure; looks like it somewhere lost the unique name for the directory

```
Cannot connect to the Docker daemon at unix:////run/docker/test/test-integration"/docker.sock. Is the docker daemon running?
```

Reply:
Perhaps we could even use a unique name for the socket itself (so that we don't have to create a directory for each one?)

Reply:
Updated, we'll see if this works with rootless.